01/1_1.playground/Contents.swift

```diff
@@ -0,0 +1,79 @@
+/*
+Exercise 1.1.  Below is a sequence of expressions. What is the result printed by the interpreter in response to each expression? Assume that the sequence is to be evaluated in the order in which it is presented.
+*/
+
+
+/*
+10
+*/
+10
+
+/*
+(+ 5 3 4)
+*/
+5 + 3 + 4
+
+/*
+(- 9 1)
+*/
+9 - 1
+
+/*
+(/ 6 2)
+*/
+6 / 2
+
+/*
+(+ (* 2 4) (- 4 6))
+*/
+(2 * 4) + (4 - 6)
+
+/*
+(define a 3)
+*/
+var a = 3
+
+/*
+(define b (+ a 1))
+*/
+var b = a + 1
+
+/*
+(+ a b (* a b))
+*/
+a + b + a + b
+
+/*
+(= a b)
+*/
+a == b
+
+/*
+(if (and (> b a) (< b (* a b)))
+b
+a)
+*/
+b > a && b < (a * b) ? b : a
+
+/*
+(cond ((= a 4) 6)
+((= b 4) (+ 6 7 a))
+(else 25))
+*/
+a == 4 ? 6 : b == 4 ? 6 + 7 + a : 25
+
+/*
+(+ 2
+(if (> b a) b a)
+)
+*/
+2 + (b > a ? b : a)
+
+/*
+(* (cond ((> a b) a)
+((< a b) b)
+(else -1))
+(+ a 1))
+*/
+
+b == a ? -1 : (a > b) ? a : b * (a + 1)
```

Comment:
This one won't produce the same values as the cond expression for all values of a & b. The initial * of a + 1 will only be carried out if a > b.  

This raises the issue of if the ternary expression is a good substitute for Lisp's `cond`. Chained ternarys are really hard to read anyway. 

One solution is to create a `cond` function in swift... the `@autoclosure` attribute gives us start, for example if we're only having one conditional statement: 

```
func cond(
    @autoclosure test: () -> Bool,
    @autoclosure expression: () -> AnyObject,
    @autoclosure fallback: (() -> AnyObject?) = nil) -> AnyObject? {

    if test() {
        return expression()
    } else {
        return fallback()
    }
}

let x = cond(false, expression: 500 + 5, fallback: "nope")
```